libs/dh/core/routing/project.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "dh-shared-routing",
```

nitpick: following the convention in all other libs, the name should contain the domain as well.

```suggestion
  "name": "dh-core-routing",
```